README.md

```diff
@@ -99,6 +99,10 @@ load(
 
 ruby_bundle(
     name = "bundle",
+    # Specify additional paths to be loaded from the gems at runtime, if any.
```

Comment:
Is that in addition to the `spec.require_paths` Gem specification?  

In other words, if a folder is part of the `spec.require_paths`, it's auto-included, and does not need to be listed in includes hash?

Reply:
Great, thanks!